pkg/security/generators/syscall_table_generator/syscall_table_generator.go

```diff
@@ -243,10 +244,5 @@ func generateStringer(inputPath, outputPath string) error {
 }
 
 func containsStringSlice(slice []string, value string) bool {
-	for _, current := range slice {
-		if current == value {
-			return true
-		}
-	}
-	return false
+	return slices.Contains(slice, value)
```

Comment:
I'm not sure we need to keep `containsStringSlice` here, might be better to update the caller directly